README

```diff
@@ -0,0 +1 @@
+
```

Comment:
the Readme file is empty

Reply:
I couldn't find the way to edit the file itself, do I have an alternative to create another new readme folder ?

Reply:
@yuisakakibara 
you already created it before
the file is `README.md`
when you open your editor, look for this file `README.md` and edit it and commit
and delete this file `README`
README